### PR TITLE
dotnet isolated fsharp templates

### PR DESCRIPTION
**Pull request by aaronpowell**
This adds project and item templates for F# under the dotnet-isolated project model.

The item templates match what originally existed for F#, so there are a few gaps still compared to the C# templates, but this brings the dotnet-isolated experience up to where we were with the in process model for F# dev.

### CODE REVIEW

**Comment by dsyme** [comment]
This is fantastic, it will be so good to have the templates supported

**Comment by mabasic** [comment]
Will this be in the v4 and in v3, or just v4?

**Comment by joshuapassos** [comment]
Anything holding this back from being merged?

**Comment by aaronpowell** [comment]
> Will this be in the v4 and in v3, or just v4?

It's possible that this could be back ported to the v3 branch, but I created it against v4 as that's the currently active dev branch.

> Anything holding this back from being merged?

Just waiting for review from someone on the Functions team. @soninaren @fabiocav would either of you have time to review this?

File: Functions.Templates/ProjectTemplate/FSharp-Isolated/Company.FunctionApp.fsproj
Comment (soninaren) [inline Functions.Templates/ProjectTemplate/FSharp-Isolated/Company.FunctionApp.fsproj:9]: might be good to update to `1.1.0-preview.1`

File: Functions.Templates/Templates/BlobTrigger-FSharp-Isolated/.template.config/template.json
Comment (soninaren) [inline Functions.Templates/Templates/BlobTrigger-FSharp-Isolated/.template.config/template.json:7]: need to add `isolated` to identity

**Review by soninaren — CHANGES_REQUESTED** [review]

**Comment by aaronpowell** [comment]
@soninaren anything more outstanding on these that you'd like me to address?

**Comment by soninaren** [comment]
@aaronpowell, Looks like your original changes were correct. I will merge and sync your template changes with c#.

**Comment by fabiocav** [comment]
@soninaren your last comment here was a while ago. Is there a reason why this hasn't been merged yet? Can we please have this in ASAP?

**Pull request merged**
